### DISCUSSION TITLE
Functional differences between interneurons

### DISCUSSION THREAD

**Discussion by rjp109** [petersenpeter/CellExplorer#65] [General]
Hello,
I was wondering if you had any insight or references regarding possible functional differences between the narrow and wide interneurons that CellExplorer classifies?
As an example, I am currently looking at differences in firing rates of putative pyr and int after administration of a drug. If they are not functionally distinct, I'm wondering whether better to just group the narrow and wide interneurons together for this. Otherwise, if there was a difference between the types of interneurons, the relevance of this finding may be difficult to explain.

Also, thank you for creating CellExplorer, it is proving very useful!

**Comment by petersenpeter** [comment]
Hi

There are definitely differences, as seen on this page: https://cellexplorer.org/publicdata/ground-truth-data/

PV cells have almost entirely narrow waveforms. SST cells are narrow but slightly wider. What we consider wide waveform interneurons are non-bursty, and typically have a lower rate than e.g. PV cells. This is true and validated for the hippocampus (CA1) and the visual cortex.

Yet, we need more ground truth data collected under similar conditions from the same brain regions to better classify this.
https://cellexplorer.org/pipeline/cell-type-classification/

I would not advise against grouping the populations, just keep in mind what effect it can have on your study. You can play with the ground truth cells included with CellExplorer to get a better understanding of this.

**Reply by rjp109** [reply]
Hello,

Okay, thank you very much for the response!
